### DISCUSSION TITLE
BPM in song select and mod select can differ

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#26402] [Q&A] [answered]
Map: https://osu.ppy.sh/beatmapsets/409214#osu/888383

With Double Time on 1.05x speed, the BPM in song select shows as 116, but the BPM in mod select shows as 115.

Song select:

![image](https://github.com/ppy/osu/assets/74088315/cec8fa6e-43ca-4ffd-8f54-b278b5f6ddd8)

Mod select:

![image](https://github.com/ppy/osu/assets/74088315/8d36a14a-e503-4fba-b331-a6e1a995a9b7)

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/26365.
